Gems/ScriptCanvas/Code/Editor/Settings.cpp

```diff
@@ -35,6 +35,7 @@ namespace ScriptCanvasEditor
 {
     namespace EditorSettings
     {
+        using namespace ScriptCanvas;;
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether